### ISSUE TITLE
Detect language

### ISSUE THREAD

**Issue by Dimsok** [zhuoqiang/goslate#1]
I can't detect languge of document. Is that possible from cmd?

**Comment by zhuoqiang** [comment]
no, the CLI interface does not support detecting language directly. Though it will try to detect the source langauge if "auto" provided as the source language (the default behavior)
